bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -337,14 +351,17 @@ type Networking struct {
 	// Defaults to a comma-delimited string of the Cluster object's spec.clusterNetwork.pods.cidrBlocks, or
 	// to "10.96.0.0/12" if that's unset.
 	// +optional
+	// +kubebuilder:validation:MaxLength=1024
```

Comment:
This should probably be changed to a list in the future reading the description

Reply:
This field is mirrored from the kubeadm API. So I think the change should be done in kubeadm first. 

Could it be that the field looks like it does because this is 1:1 what is passed into some Kubernetes component?

Reply:
Our API has to be compatible with the KubeADM API, not identical, but yes, I guess they're just exposing some command line argument. We _could_ do better

Reply:
Added a note to: https://github.com/kubernetes-sigs/cluster-api/issues/10852
Can't promise someone gets around to it for v1beta2 though